app_dart/lib/src/request_handlers/get_status.dart

```diff
@@ -28,21 +29,29 @@ class GetStatus extends RequestHandler<Body> {
   final DatastoreServiceProvider datastoreProvider;
   final BuildStatusServiceProvider buildStatusProvider;
 
-  static const String lastCommitKeyParam = 'lastCommitKey';
-  static const String branchParam = 'branch';
+  static const String kLastCommitKeyParam = 'lastCommitKey';
+  static const String kBranchParam = 'branch';
+  static const String kRepoParam = 'repo';
 
   @override
   Future<Body> get() async {
-    final String? encodedLastCommitKey = request!.uri.queryParameters[lastCommitKeyParam];
-    final String branch = request!.uri.queryParameters[branchParam] ?? 'master';
+    final String? encodedLastCommitKey = request!.uri.queryParameters[kLastCommitKeyParam];
+    final String repoName = request!.uri.queryParameters[kRepoParam] ?? config.flutterSlug.name;
+    final RepositorySlug slug = RepositorySlug('flutter', repoName);
+    final String branch = request!.uri.queryParameters[kBranchParam] ?? 'master';
```

Comment:
Engine is now with `main` as default.

Reply:
Added! I made `config.defaultBranch` into `Config.defaultBranch(RepositorySlug)` to correctly return the default branch everywhere